packages/@best/api-db/src/sql/sqlite/migrations/002-organization.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+*/
+
+export const up = `
+CREATE TABLE organizations (
+    id SERIAL PRIMARY KEY,
+    name character varying(100) NOT NULL,
+    created_at timestamp without time zone NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE UNIQUE INDEX org_unique_name ON organizations(name text_ops);
+
+ALTER TABLE projects
+ADD COLUMN organization_id INTEGER;
```

Comment:
make sure to add the key reference. Same for the postgres migration too, should be something like this:
```suggestion
ADD COLUMN organization_id INTEGER NOT NULL REFERENCES organizations(id) ON DELETE CASCADE
```

Reply:
@treywashington we cannot add references to organizations when there is already data contains in existing projects. It will throw an error, in this case first we need to update the data for the existing project with the organization id that it belongs to and then we add reference key to organization table.